packages/protoc-gen-connect-query/snapshots/buf.gen.yaml

```diff
@@ -14,3 +18,10 @@ plugins:
       - target=ts+dts+js
       - import_extension=none
       - ts_nocheck=false
+  - plugin: connect-query
+    path: ../bin/protoc-gen-connect-query
+    out: gen_cjs
+    opt:
+      - target=dts+js
+      - import_extension=.js
+      - js_import_style=legacy_commonjs
```

Comment:
Shouldn't we set `ts_nocheck=false` here as well, same as we do above?

The snapshots are great, but I find them a bit difficult to follow. Here's the tree:

```
.
├── buf.gen.yaml
├── gen
│   └── connectrpc
│       └── eliza
│           └── v1
│               ├── eliza-ElizaService_connectquery.d.ts
│               ├── eliza-ElizaService_connectquery.js
│               ├── eliza-ElizaService_connectquery.ts
│               └── eliza_pb.ts
└── gen_cjs
    └── connectrpc
        └── eliza
            └── v1
                ├── eliza-ElizaService_connectquery.d.ts
                ├── eliza-ElizaService_connectquery.js
                └── eliza_pb.ts
```

In `gen`, it's a bit odd that we have both ts and d.ts files, and in `gen_js`, we mix js and ts. This may work with our linter, but you would never use this setup in practice, right?

WDYT about making the snapshots more realistic by generating three use cases: ts, js, and cjs?

```yaml
version: v1
plugins:
  # js
  - plugin: es
    out: gen_js
  - plugin: connect-query
    path: ../bin/protoc-gen-connect-query
    out: gen_js

  # cjs
  - plugin: es
    out: gen_cjs
    opt:
      - js_import_style=legacy_commonjs
  - plugin: connect-query
    path: ../bin/protoc-gen-connect-query
    out: gen_cjs
    opt:
      - js_import_style=legacy_commonjs
      - ts_nocheck=false # we only set this for our tests

  # ts
  - plugin: es
    out: gen_ts
    opt:
      - target=ts
  - plugin: connect-query
    path: ../bin/protoc-gen-connect-query
    out: gen_ts
    opt:
      - target=ts
      - ts_nocheck=false # we only set this for our tests
```

Reply:
Yeah, that does sound like a better option. I'll make the change.